src/web/src/components/AboutFooter.tsx

```diff
@@ -3,8 +3,7 @@ import { Box, Divider, Grid, Typography, useMediaQuery } from '@material-ui/core
 import GitHubIcon from '@material-ui/icons/GitHub';
 import GitHubContributorCard from './GitHubContributorCard';
 import Logo from './Logo';
-
-const slackLogo = '/slack-logo.svg';
```

Comment:
Can we delete this file if you're not using it here (i.e., is it used anywhere else?  If not, remove it).

Reply:
I'll look into it and delete it if we don't need it.